src/styles/main.scss

```diff
@@ -76,12 +76,6 @@ a {
     text-decoration: none;
   }
 
-  &>* {
-    // to prevent the child elements inside a tag from registering
-    // as the target in event listeners (for analytics tracking)
-    pointer-events: none;
-  }
```

Comment:
@tshen815 I am assuming I can't just remove all of it but we should probably not target every `a` children either?

Reply:
agreed, Im not sure when this was set in place but targeting every `a` seems risky